app/src/main/kotlin/com/hippo/ehviewer/client/parser/GalleryDetailParser.kt

```diff
@@ -76,7 +76,7 @@ object GalleryDetailParser {
     private val PATTERN_NEWER_DATE = Regex(", added (.+?)<br />")
     private val PATTERN_FAVORITE_SLOT =
         Regex("/fav.png\\); background-position:0px -(\\d+)px")
-    private val EMPTY_GALLERY_TAG_GROUP_ARRAY = arrayOf<GalleryTagGroup>()
+    private val EMPTY_GALLERY_TAG_GROUP_ARRAY = emptyList<GalleryTagGroup>()
     private val EMPTY_GALLERY_COMMENT_ARRAY = GalleryCommentList(emptyList(), false)
```

Comment:
Rename these two to match their types?

Reply:
I inlined it since emptyList() always return a singleton

Reply:
The empty GalleryCommentList object?

Reply:
EMPTY_GALLERY_TAG_GROUP_ARRAY

Reply:
Yeah I mean rename the GalleryCommentList object.

Reply:
Done